cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/instance/StopStartDownscaleStopInstancesRequest.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.cloudbreak.cloud.event.instance;
+
+import java.util.List;
+
+import com.sequenceiq.cloudbreak.cloud.context.CloudContext;
+import com.sequenceiq.cloudbreak.cloud.event.resource.CloudStackRequest;
+import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
+import com.sequenceiq.cloudbreak.cloud.model.CloudInstance;
+import com.sequenceiq.cloudbreak.cloud.model.CloudStack;
+
+public class StopStartDownscaleStopInstancesRequest<T> extends CloudStackRequest<T> {
```

Comment:
maybe it would be better to use StopStartDownscaleStopInstancesResult instead of T

Reply:
Will change this.

Reply:
Fixed via CB-15827